### ISSUE TITLE
Trouble restoring backups on different phones

### ISSUE THREAD

**Issue by 01zero** [inorichi/tachiyomi#2591]
**App version: r1719**

**Android version:6.0.1 to 9**

**Issue/Request:** Hi, I recently changed my phone from a Samsung Galaxy J2 Pro/Prime running Android 6.0.1 to a Xiaomi Redmi Note 8 running Android 9. I figured I'd do a backup of my library and then restore that file on my new phone, but when I go to >Settings>Backup>Restore Backup and select my backup file, I receive the following message: 
![Screenshot_2020-02-13-20-46-22-250_eu kanade tachiyomi debug](https://user-images.githubusercontent.com/61032965/74489554-5e2f9080-4ea4-11ea-9fd4-97d01aa5d213.jpg)

**Steps to reproduce (if applicable)**

 1.Open tachiyomi on Android 6.0.1, go to settings and do a backup.
 2.Install the same version (r1719) on Android 9, along with all previous extensions.
 3.Go to Settings>Backup>Restore Backup

**I apologize if this was posted on the wrong page/category, this is my first time and I don't intend to break any rules**

Edit: Here's the backup file
http://www.mediafire.com/file/yrz8jrv0dhu0lnz/tachiyomi_2020-02-13_20-42.json/file

**Comment by arkon** [comment]
It'd help if you could post the file somewhere, if you're comfortable with that.

**Comment by 01zero** [comment]
> It'd help if you could post the file somewhere, if you're comfortable with that.

Ok, I'll edit my post and link it. Thanks for the reply

**Comment by arkon** [comment]
The file seems to be invalid for some reason. You could try doing another backup/restore.

**Comment by 01zero** [comment]
> The file seems to be invalid for some reason. You could try doing another backup/restore.

Ok, after a bit of trial and error the problem is sort of solved. In the create backup window I started unchecking the boxes that were available, until I finally got something valid. I had to uncheck all, leaving me only with the Manga one (which is uncheckable). After that I restored the backup succesfully. I had to mark manually the chapters that I had read previously, but it's still better than nothing.
Thanks for the contribution
